app/views/catalog/_component_summary.html.erb

```diff
@@ -14,6 +14,8 @@
     </div>
 
   </div>
-
+  <% if document.digital_objects.present? %>
+    <%= render(partial: "arclight_viewer_default") %>
```

Comment:
There should be a follow up for dealing with un-styled digital object links. E.g. `catalog/MC221_c0118`